src/side-nav/side-nav-divider.jsx

```diff
@@ -0,0 +1,29 @@
+let React = require('react/addons');
+let StylePropable = require('../mixins/style-propable');
+let MenuDivider = require('../menus/menu-divider');
+
+let SideNavDivider = React.createClass({
+
+  mixins: [StylePropable],
+
+  contextTypes: {
+    muiTheme: React.PropTypes.object,
+  },
+
+  render() {
+    let {
+      style,
+      ...other,
+    } = this.props;
+
+    let mergedStyles = this.mergeAndPrefix({
+      /*TODO*/
```

Comment:
So this component basically should just add some styles to MenuDivider, but for now it does'n do even that. Do you think this component is really needed?